### ISSUE TITLE
Having issues adding motion and temp/humidity sensor

### ISSUE THREAD

**Issue by iamjmax** [mdaskalov/homebridge-tasmota-zbbridge#26]
First of all, great plug in! I managed to get my contact sensor working correctly (working in home bridge/homekit) following your example, but I can't seem to get my sonoff motion sensor or temp/humidty sensor to work properly. I'm unsure of the sensorvaluecharacteristics or sensorValuePath that I should be using for these. Any help would be greatly appreciated!

**Comment by mdaskalov** [comment]
If we’re talking about the SZNB-02 you have to configure 2 ZbDevices of type `sensor` and configure them as follows:
1) select `HumiditySensor` for sensorService, enter `CurrentRelativeHumidity` as sensorCharacteristic and `Humidity` for sensorValuePath.
2) select `TemperatureSensor` for sensorService, enter `CurrentTemperature` as sensorCharacteristic and `Temperature` as sensorValuePath.

For the motion sensor configure another ZbDevice. Select `OccupancySensor` for sensorService, enter `OccupancyDetected` as sensorCharacteristic and `Occupancy` as sensorValuePath (not sure about this).

The configuration json will be (you have to change the `addr` accordingly):
```
        {
            "addr": "0x00124B001F841E41",
            "type": "sensor",
            "name": "Humidity Sensor",
            "sensorService": "HumiditySensor",
            "sensorCharacteristic": "CurrentRelativeHumidity",
            "sensorValuePath": "Humidity"
        },
        {
            "addr": "0x00124B001F841E41",
            "type": "sensor",
            "name": "Temperature Sensor",
            "sensorService": "TemperatureSensor",
            "sensorCharacteristic": "CurrentTemperature",
            "sensorValuePath": "Temperature"
        },
        {
            "addr": "0x00124B0026B684E4",
            "type": "sensor",
            "name": "Motion Sensor",
            "sensorService": "OccupancySensor",
            "sensorCharacteristic": "OccupancyDetected",
            "sensorValuePath": "Occupancy"
        }
```
I recommend using the hardware address (IEEEAddr) in `addr` as this makes things easier when you have to re-pair the sensor. The plugin maintains the mapping internally (Tasmota uses the short address almost everywhere). Use `zbinfo` to find out the hardware address.

The `sensorValuePath` parameter is be the name of the value property in the object where the `Device` property is set. Depending on the options [SetOption83](https://tasmota.github.io/docs/Features/#setoption83), [SetOption89](https://tasmota.github.io/docs/Features/#setoption89), [SetOption100](https://tasmota.github.io/docs/Features/#setoption100), [SetOption101](https://tasmota.github.io/docs/Features/#setoption101) this object could be found in a different place of the ZbReceived response. This doesn’t matter as the object always contains the same properties.

For example if you receive:
```
17:08:14.221 RSL: SENSOR = {"ZbReceived":{"0x2916":{"Device":"0x2916","Temperature":25.72,"Humidity":47.73,"Endpoint":1,"LinkQuality":116}}}
``` 

The object where the `sensorValuePath` is located would be:
```
{
  "Device":"0x2916",
  "Temperature":25.72,
  "Humidity":47.73,
  "Endpoint":1,
  "LinkQuality":116
}
```

**Comment by iamjmax** [comment]
Thanks for the quick reply! I have all working except the humidity sensor. It doesn't show any data. Below is from my config file. Any ideas?

"addr": "0x00124B00251CA8A9",
                    "type": "sensor",
                    "name": "Humidity",
                    "sensorService": "HumiditySensor",
                    "sensorCharacteristic": "CurrentRelativeHumidity",
                    "sensorValuePath": "Humidity"


And from zbinfo in tasmota--

4:06:53.024 MQT: tele/Tasmota/SENSOR = {"ZbInfo":{"0x8653":{"Device":"0x8653","Name":"Humidity","IEEEAddr":"0x00124B00251CA8A9","ModelId":"TH01","Manufacturer":"eWeLink","Endpoints":[1],"Config":["T01"],"Temperature":20.75,"Humidity":53.91,"Reachable":true,"BatteryPercentage":100,"BatteryLastSeenEpoch":1664715470,"LastSeen":231,"LastSeenEpoch":1664715782,"LinkQuality":120}}}

**Comment by mdaskalov** [comment]
The configuration looks fine. 

The problem is that you have twice the same accessory from homebridge point of view - the accessory unique-id is generated using addr and type. 

Try using the short-address for one of the sensors.

I will have to change this in the next release...

**Comment by iamjmax** [comment]
Thanks, I tried using the short address as well for both, same issue. I also changed one to short, and one to long with the same result.

**Comment by mdaskalov** [comment]
Will have to check this. Does it work if you configure only humidity sensor?

**Comment by iamjmax** [comment]
I have not tried that. Will do now.

**Comment by iamjmax** [comment]
Can confirm, if I remove the temp sensor accessory, the humidity starts to work.

**Comment by mdaskalov** [comment]
okay, then please wait for the next release (probably later today).

**Comment by iamjmax** [comment]
Thanks again!

**Comment by mdaskalov** [comment]
Just published release v1.4.4 where this should be solved. You can now use the hardware address for both sensors.
Please confirm.
